src/containers/login/LoginForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { FormGroup, FormControl, ControlLabel, Button } from 'react-bootstrap';
+
+const LoginForm = () => (                              
+     <div>
+          <form>
+               <h2>Login</h2>
+               <FormGroup controlId="formValidationSuccess1" validationState="success">
```

Comment:
I suggest you avoid default react names and also follow a particular naming standard. So you could name this controlId `email-group` for example.